idd/Energy+.idd.in

```diff
@@ -7604,6 +7604,61 @@ MaterialProperty:VariableThermalConductivity,
        \units W/m-K
        \type real
 
+MaterialProperty:VariableAbsorptance,
+  A1 , \field Name
+       \required-field
+       \type alpha
+       \note The name of the dynamic coating material with variable thermal or solar absorptance.
+  A2 , \field Reference Material Name
+       \required-field
+       \type object-list
+       \object-list MaterialName
+       \note Regular Material Name to which the additional properties will be added.
+       \note this the material name for the basic material properties.
+  A3 , \field Control Signal
+       \required-field
+       \type choice
+       \key SurfaceTemperature
+       \key SurfaceReceivedSolarRadiation
+       \key SpaceHeatingCoolingMode
+       \key Scheduled
+       \default SurfaceTemperature
+       \note the variable that drives the change in thermal/solar absorptance
+  A4 , \field Thermal Absorptance Function Name
+       \note A Curve:* or Table:Lookup object encoding the relationship between
+       \note the control signal value and the surface thermal absorptance.
+  A5 , \field Thermal Absorptance Schedule Name
+       \note only used when Control Signal = "Scheduled"
+  A6 , \field Solar Absorptance Function Name
+       \note A Curve:* or Table:Lookup object encoding the relationship between
+       \note the control signal value and the surface solar absorptance.
+  A7 ; \field Solar Absorptance Schedule Name
+       \note only used when Control Signal = "Scheduled"
+
+MaterialProperty:VariableSolarAbsorptance,
```

Comment:
@yujiex This object should be deleted, correct?

Reply:
Oh, I forgot to take that out of idd. We only have one object now, MaterialProperty:VariableAbsorptance

Reply:
just removed it

Reply:
Was this comment from an old object?  It is showing up on the `MaterialProperty:HeatAndMoistureTransfer:Settings` object now, but I don't think that object would be deleted here.